### ISSUE TITLE
Why doesn't my neo-tree panel show relative numbers like a normal buffer?

### ISSUE THREAD

**Issue by jonahfang** [mluders/comfy-line-numbers.nvim#2]
It seems that the neo-tree panel cannot use the effects of your plugin.

**Comment by mluders** [comment]
Thanks! Appreciate you bringing this up.

I'll take a look asap.

**Comment by mluders** [comment]
This seems to be the default behavior with Neotree.

To demonstrate this, select the Neotree buffer and run `:set signcolumn=yes`. The sign column then appears, and Comfy Line Numbers are displayed as expected.

I found a [Reddit thread](https://www.reddit.com/r/neovim/comments/1bdxksy/how_to_automatically_perform_action_when_entering/) that describes a similar issue. Following the advice in the thread, you probably need to create an autocommand that executes `:set signcolumn=yes` upon opening Neotree.

When testing Neotree, I did discover an unrelated bug in my code. I just pushed a fix: https://github.com/mluders/comfy-line-numbers.nvim/commit/946912de5aa5a07b99928a45a60265029c8d97b8

Hopefully this is helpful!
